EscapeApp/app/src/main/java/com/context/kroket/escapeapp/minigames/D_Lock.java

```diff
@@ -180,7 +197,7 @@ private void rotate(int alpha) {
   @Override
   protected void onStart() {
     super.onStart();
-
+    tv = (TextView) findViewById(R.id.locktext);
     Intent i = new Intent(this, ConnectionService.class);
```

Comment:
Please use "Intent intent" instead of "Intent i".
The variable name "tv" isn't descriptive. Please consider calling this lockText or lockField or something.